tests/basilisp/test_core_fns.lpy

```diff
@@ -2156,7 +2156,19 @@
 
 (deftest pr-test
   (testing "is dynamic"
-    (is (= '(1) (binding [pr (fn [& more] more)] (pr 1))))))
+    (is (= '(1) (binding [pr (fn [& more] more)] (pr 1)))))
+
+  (testing "with *print-namespace-maps*"
+    (are [res m b] (contains? res (binding [*print-namespace-maps* b] (pr-str m)))
```

Comment:
Can you add some cases with `*print-meta*` since that's a separate branch.

Reply:
Tests added, thanks